### ISSUE TITLE
Add "Initial Execution ID"  as a context variable for  Job Retries  jobs

### ISSUE THREAD

**Issue by sebastianbello** [rundeck/rundeck#3207]
Issue type: Enhancement Request

**My Rundeck detail**

* Rundeck version: 2.10.6
* install type: (rpm,deb,war,launcher?)
* OS Name/version: 
* DB Type/version: mysql? postgres? h2?..

**Expected Behavior**
When a job execution is Retried it would be useful to know the ID of the Initial Execution as a Context Variable. 
Currently, the existing (related) context variables are:
`job.id`  - ID of the Job
`job.execid` - ID of the current Execution
`job.retryAttempt` - A number indicating the attempt, if this execution is a retry.
`job.wasRetry`  - _true_ if this execution is a retry, otherwise, _false_

Adding the "Initial Execution ID" as a context variable would be a good enhancement.

**How to Reproduce**

Basically we currently have `job.retryAttempt`  and `job.wasRetry` but can't tell the `"initial" ExecId` ( initial execution)  without performing a query, by `job.id` and relate with `job.retryAttempt` , assuming `job.retryAttempt` =0 would be the "parent" ejecution and `job.retryAttempt` > 0 for the retries.

JobID=816db445-69ba-4fa5-90bd-cb720fe646b8 RetryAttempt=0 WasRetry=false ExecId=52355 – > "initial" or "parent" job
JobID=816db445-69ba-4fa5-90bd-cb720fe646b8 RetryAttempt=2 WasRetry=true ExecId=52357 – > 2nd retry of job
JobID=816db445-69ba-4fa5-90bd-cb720fe646b8 RetryAttempt=4 WasRetry=true ExecId=52359 – > 4nd retry of job
